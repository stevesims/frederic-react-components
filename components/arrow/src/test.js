import React from 'react';
import { shallow } from 'enzyme';

import Component from '.';

describe('Arrow', () => {
  let wrapper;

  it('renders without crashing', () => {
    wrapper = shallow(<Component />);
  });

  it('sets fill color on polygon', () => {
    wrapper = shallow(<Component fill="red" />);
    expect(wrapper.find('polygon[fill="red"]').length).toBe(1);
  });

  it('matches snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
