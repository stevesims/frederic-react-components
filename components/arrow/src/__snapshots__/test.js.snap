// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Arrow matches snapshot 1`] = `
<svg
  height="20px"
  viewBox="0 0 6 4"
  width="20px"
>
  <polygon
    fill="red"
    points="3 4 6 1 0 1"
  />
</svg>
`;
